egs/swbd/s5c/RESULTS

```diff
@@ -109,6 +109,20 @@ exit 0
 %WER 11.8 | 1831 21395 | 89.4 7.0 3.6 1.3 11.8 47.8 | exp/nnet3/lstm_ld5_sp/decode_eval2000_sw1_fsh_fg/score_13_0.0/eval2000_hires.ctm.swbd.filt.sys
 %WER 13.2 | 1831 21395 | 88.3 8.0 3.7 1.5 13.2 49.5 | exp/nnet3/lstm_ld5_sp/decode_eval2000_sw1_tg/score_11_0.0/eval2000_hires.ctm.swbd.filt.sys
 
+# these are results with nnet3 LSTMs cell_dim=1280, recurrent_dim=384, lstm_delay=-1 -2 -3, label_delay=5 num_params=25010228 (8 epoch training on speed-perturbed
+# and volume perturbed data)
+%WER 15.20 [ 7477 / 49204, 1031 ins, 1912 del, 4534 sub ] exp/nnet3/lstm_1280_384_123_ld5_sp/decode_train_dev_sw1_fsh_fg/wer_12_0.0
```

Comment:
Just give results for eval set.
Replace the default parameter settings for the LSTM with the those that give you best result. Remove the old LSTM results. The RESULTS file should have results that can be reproduced by running the local/nnet3/run_lstm.sh script.